sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableEntityValueBinderTests.cs

```diff
@@ -0,0 +1,336 @@
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.Guid, property.PropertyType);
+            Assert.Null(property.GuidValue);
+        }
+
+        [Test]
+        public void DeepClone_IfInt32_PreservesValue()
+        {
+            // Arrange
+            int expected = 123;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(expected));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.Int32, property.PropertyType);
+            Assert.AreEqual(expected, property.Int32Value);
+        }
+
+        [Test]
+        public void DeepClone_IfInt32Null_PreservesValue()
+        {
+            // Arrange
+            int? input = null;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(input));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.Int32, property.PropertyType);
+            Assert.Null(property.Int32Value);
+        }
+
+        [Test]
+        public void DeepClone_IfInt64_PreservesValue()
+        {
+            // Arrange
+            long expected = 123;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(expected));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.Int64, property.PropertyType);
+            Assert.AreEqual(expected, property.Int64Value);
+        }
+
+        [Test]
+        public void DeepClone_IfInt64Null_PreservesValue()
+        {
+            // Arrange
+            long? input = null;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(input));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.Int64, property.PropertyType);
+            Assert.Null(property.Int64Value);
+        }
+
+        [Test]
+        public void DeepClone_IfString_PreservesValue()
+        {
+            // Arrange
+            string expected = "abc";
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(expected));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.String, property.PropertyType);
+            Assert.AreSame(expected, property.StringValue);
+        }
+
+        [Test]
+        public void DeepClone_IfStringNull_PreservesValue()
+        {
+            // Arrange
+            string input = null;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(input));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.String, property.PropertyType);
+            Assert.Null(property.StringValue);
+        }
+
+        private class SimpleTableEntity
```

Comment:
Is this used?

Reply:
I didn't change this file. I initially removed it and then added it back. It's still excluded from the project.